src/plugins/guided_onboarding/public/components/guide_panel.tsx

```diff
@@ -198,7 +206,13 @@ export const GuidePanel = ({ api, application }: GuidePanelProps) => {
             </EuiButtonEmpty>
 
             <EuiTitle size="m">
-              <h2>{guideConfig?.title}</h2>
+              <h2 data-test-subj="guideTitle">
+                {isGuideReadyToComplete
```

Comment:
I debated whether it would be better to create a separate component for the "completed" state instead of this conditional logic throughout. I decided to leave as-is for now, but happy to revisit.